src/FunctionApps/python/IntakePipeline/fhir.py

```diff
@@ -0,0 +1,9 @@
+from typing import Iterator
+
+
+def read_fhir_bundles() -> Iterator[dict]:
+    yield {}
+
+
+def upload_bundle_to_fhir_server(bundle: dict) -> None:
```

Comment:
If a `bundle` is a bunch of HL7 messages that was converted to FHIR, won't that put ndjson rather than just a json/dict object? I could be wrong, but that parameter might need to be a list of dicts.

Reply:
nope - it's not ndjson, it's just json

Reply:
Huh. So what does that json look like? If a bundle of HL7 messages is just a list of dicts conforming to some specification, the `convert_to_fhir` function should turn that into a list of dicts conforming to a different standard as I understood it. If it becomes json, how does that work with keys and mappings?

Reply:
Yeah, ndjson is a different concept from Bundles.  A bundle is actually its own FHIR resource type that includes a collection of other resources.  

Here's an example of a FHIR bundle:
https://www.hl7.org/fhir/diagnosticreport-example-f001-bloodexam.json.html
and the spec...
https://www.hl7.org/fhir/bundle.html